### PR TITLE
Added missing getter for dataTypePriorityCode

### PR DESCRIPTION
**Pull request by onursumer**
# What? Why?
Fix https://github.com/cBioPortal/cbioportal/issues/2962.

# Checks
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Airbnb React/JSX Style guide](https://github.com/airbnb/javascript/tree/master/react).
- [ ] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)

### CODE REVIEW

File: src/shared/components/query/QueryStore.ts [hunk 1]
```diff
@@ -542,6 +543,11 @@ export class QueryStore
 		}
 	}
 
+	get dataTypePriorityCode(): '0'|'1'|'2'
```
Comment (adamabeshouse) [inline src/shared/components/query/QueryStore.ts]: `@computed`?

Comment (onursumer) [inline src/shared/components/query/QueryStore.ts]: yea, makes sense!

File: src/shared/components/query/QueryStore.ts [hunk 2]
```diff
@@ -513,7 +513,8 @@ export class QueryStore
 
 	// DATA TYPE PRIORITY
 
-	private getDataTypePriorityCode(dataTypePriority:{mutation: boolean, cna: boolean}) {
+	private getDataTypePriorityCode(dataTypePriority:{mutation: boolean, cna: boolean}): '0'|'1'|'2'
```
Comment (alisman) [inline src/shared/components/query/QueryStore.ts]: just curious but is there a reason we use strings for integers?

Comment (alisman) [inline src/shared/components/query/QueryStore.ts]: lets rename this to calculateDataType...

Comment (onursumer) [inline src/shared/components/query/QueryStore.ts]: good point, no idea, I just added the return type for clarity :)

Comment (onursumer) [inline src/shared/components/query/QueryStore.ts]: sounds good

**Pull request merged**